stable/nginx-ingress/templates/controller-configmap.yaml

```diff
@@ -10,8 +10,11 @@ metadata:
     release: {{ .Release.Name }}
   name: {{ template "nginx-ingress.controller.fullname" . }}
 data:
-{{- if .Values.controller.headers }}
-  proxy-set-headers: {{ .Release.Namespace }}/{{ template "nginx-ingress.fullname" . }}-custom-headers
+{{- if .Values.controller.addHeaders }}
```

Comment:
Same as previous comment. This would be a breaking change.

Reply:
controller.headers and controller.proxySetHeaders now flex in the proxyheaders-configmap.yaml template file.